app/views/submissions/show.erb

```diff
@@ -6,6 +6,9 @@
     </h1>
 
     <div class="pull-right submission-toolbar hidden-xs">
+      <% if submission.user_exercise.help_requested? %>
+        <span class="label label-danger">Requested help</span>
```

Comment:
Ok, I've looked at the `public/sass/base/themes/exercism.scss` file, and it looks like danger **is** appropriate to use here, since it's used by so many other elements and I don't want to inflict a css refactoring onto you.